scripts/verify_crop_recommendations.py

```
@@ -0,0 +1,356 @@
+#!/usr/bin/env python3
+"""
+🔬 Проверка точности рекомендаций по культурам
+Проверяет все значения влажности, pH, EC и NPK на соответствие научным источникам
+
+Author: JXCT Soil Sensor Project
+Date: 2025
+"""
+
+import re
+import json
+from typing import Dict, List, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)